webapp/channels/src/utils/a11y_controller.ts

```diff
@@ -810,14 +810,14 @@ export default class A11yController {
             break;
         case isKeyPressed(event, Constants.KeyCodes.ENTER):
             this.enterKeyIsPressed = true;
-            break;
-        case isKeyPressed(event, Constants.KeyCodes.SPACE):
```

Comment:
Those failing tests are likely caused by this change. We can't change how the space bar is handled here because it affects all buttons in the UI.

Instead, I think you'd want to add an `onKeyDown` handler to the category header button, and if the space bar is pressed in that handler, you can call `e.stopPropagation()` there to prevent it from making it this far. If `dragHandleProps` includes its own `onKeyDown`, you'd have to call that then